bootstrap.d/app-shells.yml

```diff
@@ -48,3 +48,37 @@ packages:
       - args: ['cp', '@SOURCE_ROOT@/extrafiles/profile', '@THIS_COLLECT_DIR@/etc']
       - args: ['cp', '@SOURCE_ROOT@/extrafiles/bash.bashrc', '@THIS_COLLECT_DIR@/etc']
       - args: ['ln', '-sf', 'bash', '@THIS_COLLECT_DIR@/usr/bin/sh']
+
+  - name: fish
+    metadata:
+      summary: Friendly interactive SHell
+      description: Smart and user-friendly portable command line shell.
+      spdx: 'GPL-2'
+      website: 'https://fishshell.com/'
+      maintainer: "Matt Taylor <mstaveleytaylor@gmail.com>"
+    architecture: '@OPTION:arch@'
+    source:
+      subdir: 'ports'
+      git: 'https://github.com/fish-shell/fish-shell.git'
+      tag: '3.4.1'
+      version: '3.4.1'
+    tools_required:
+      - system-gcc
+      - host-cmake
+    pkgs_required:
```

Comment:
Missing dependency on `mlibc`, I'm pretty sure it depends on the libc.

Reply:
Also seems like one can use `libintl` for i18n support, we can specify the dependency for later use.